outlines/text/generate/regex.py

```diff
@@ -129,9 +129,10 @@ def create_proposal(
             self.pstates = new_pstates
 
         masks = []
+        mask_shape = (len(logits[0]),)
```

Comment:
While this is going to work, I'd like to have an explanation as to why the vocab size does not correspond to the logit size for these models before merging. Do you have an idea?

Reply:
Well I can't say for sure on how vocabs are created in pythia models, may be due to how long these models are being trained or these vocab's are shared between different models who are trained for large set of tokens, I just found that we creating a mask for logits with vocab size and since vocab size's are different at each level incase of pythia/GTPNeoXTokenizer
I thought of creating a mask directly with logit size,

Reply:
on the EleuterAI discord release channel "the embedding dim is rounded  up to the nearest multiple of 128 * model-parallel-size  for training efficiency" in response to this question.  the higher token ids are there but don't mean anything.  I think this means that @xaviruvpadhiyar98's approach is correct; rounding 50254 up to nearest multiple of 128 gives you 50304.

Reply:
Thanks @maedoc for confirming..